scripts/variantstore/wdl/extract/import_gvs.py

```diff
@@ -397,10 +393,6 @@ def convert_array_with_id_keys_to_dense_array(arr, ids, drop=[]):
 
         vd = vd.drop('allele_NO', 'allele_YES', 'allele_is_snp', 'allele_OK')
 
-        if unphase:
-            vd = vd.annotate_entries(LGT=vd.LGT.unphase())
```

Comment:
Do we recall why we did this originally?

Reply:
yes--we didn't include any phasing information in the Delta release, but... GTs still were still coming out of the GVCFS and thus into BigQuery with a phased GT (either a | or a / ) so then they were showing up in the avros that way too. We wanted to reset the GTs (well LGTs) fully to avoid confusion